code/modules/tgui/tgui_input_list.dm

```diff
@@ -94,12 +96,15 @@
 	var/timeout
 	/// Boolean field describing if the tgui_modal was closed by the user.
 	var/closed
+	/// String field for the theme to use
+	var/ui_theme
```

Comment:
we should eventually get around to having these as defines
something for another pr

Reply:
yeah that kind of thing would work but is probably out of scope for this pr

alternatively put it in the general tgui defines file - I presume there is one